client/local/local.go

```diff
@@ -414,6 +414,24 @@ func (lc *Client) TailDaemonLogs(ctx context.Context) (io.Reader, error) {
 	return res.Body, nil
 }
 
+// TailBusEvents returns a stream the Tailscale bus events as they arrive.
+// Close the context to stop the stream.
+func (lc *Client) TailBusEvents(ctx context.Context) (io.Reader, error) {
```

Comment:
I think we should make this an `io.ReadCloser` so the response body gets closed as is required by the HTTP API. (Also to note: Even if ctx is closed, that cancels the request but I'm not sure that is sufficient to cancel the request once the initial reply has come back and we're chewing on the chunks)